### ISSUE TITLE
Doesn't work on Ubuntu 22.04

### ISSUE THREAD

**Issue by maxpain** [saul/csgo-cvar-unhide#26]
Hello. This plugin doesn't work on Ubuntu 22.04:

<img width="437" alt="image" src="https://user-images.githubusercontent.com/5875582/206912435-6ccfa33d-bb3a-457f-ad3e-7127b59b807b.png">

<img width="810" alt="image" src="https://user-images.githubusercontent.com/5875582/206912457-41e9f6c4-88fe-4f98-9d01-b0b4f181add7.png">

**Comment by saul** [comment]
Please can you run `ldd cvar-unhide.so` and paste the output here as a comment?

**Comment by maxpain** [comment]
```bash
root@csgo-59c8b769-bd488:~/.steam/steam/steamapps/common/Counter-Strike Global Offensive/csgo/addons# ldd cvar-unhide.so
        linux-gate.so.1 (0xf7f04000)
        libstdc++.so.6 => /lib/i386-linux-gnu/libstdc++.so.6 (0xf7c7d000)
        libgcc_s.so.1 => /lib/i386-linux-gnu/libgcc_s.so.1 (0xf7c56000)
        libtier0.so => not found
        libc.so.6 => /lib/i386-linux-gnu/libc.so.6 (0xf7a21000)
        libm.so.6 => /lib/i386-linux-gnu/libm.so.6 (0xf7919000)
        /lib/ld-linux.so.2 (0xf7f06000)
```

**Comment by saul** [comment]
Are there any other errors in the console? Remember you need to run CSGO with `-insecure` as the engine won't load plugins with VAC enabled.

**Comment by maxpain** [comment]
Maybe the problem is caused by this?
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/5875582/206912710-f4ee8ede-40df-477f-858e-bbda5594263a.png">

**Comment by maxpain** [comment]
> Remember you need to run CSGO with -insecure as the engine won't load plugins with VAC enabled.

Yes, I started with `-insecure`

**Comment by saul** [comment]
> Maybe the problem is caused by this?

No, this is expected as `libtier0.so` is in a different directory.

I don't have a Linux VM handy to test right now but I will try to reproduce the issue soon.

**Comment by maxpain** [comment]
> Are there any other errors in the console?
<img width="997" alt="Screenshot 2022-12-11 at 19 35 25" src="https://user-images.githubusercontent.com/5875582/206912967-e3f2eea3-2461-4dfe-9bec-1f05b447b159.png">
<img width="1015" alt="Screenshot 2022-12-11 at 19 35 52" src="https://user-images.githubusercontent.com/5875582/206912970-650d7c05-bafd-49ab-867e-c8efac594243.png">

**Comment by maxpain** [comment]
Actually, I run CS:GO in a container with GPU in Kubernetes on GKE.
I use `nvidia/cuda:11.8.0-base-ubuntu22.04` as base image

I install these packages:
```bash
apt install -y \
    xvfb \
    x11vnc \
    pciutils \
    net-tools \
    lsof \
    locales \
    pulseaudio \
    steam \
    steamcmd
```

Maybe I need some additional libs, idk.

Launch options:
```
-vulkan -windowed -insecure
```

Everything works great, except for this plugin, which will be helpful for my case :).

**Comment by maxpain** [comment]
Any updates?
